charts/kube-prometheus-stack/templates/prometheus/rules-1.14/kubernetes-storage.yaml

```diff
@@ -35,7 +35,7 @@ spec:
 {{- if .Values.defaultRules.additionalRuleGroupAnnotations.kubernetesStorage }}
 {{ toYaml .Values.defaultRules.additionalRuleGroupAnnotations.kubernetesStorage | indent 8 }}
 {{- end }}
-        description: The PersistentVolume claimed by {{`{{`}} $labels.persistentvolumeclaim {{`}}`}} in Namespace {{`{{`}} $labels.namespace {{`}}`}} on Cluster {{`{{`}} $labels.cluster {{`}}`}} is only {{`{{`}} $value | humanizePercentage {{`}}`}} free.
+        description: The PersistentVolume claimed by {{`{{`}} $labels.persistentvolumeclaim {{`}}`}} in Namespace {{`{{`}} $labels.namespace {{`}}`}} {{`{{`}} with $labels.cluster -{{`}}`}} on Cluster {{`{{`}} . {{`}}`}} {{`{{`}}- end {{`}}`}} is only {{`{{`}} $value | humanizePercentage {{`}}`}} free.
```

Comment:
Is this change intentional? Looks like it's reverting changes applied on https://github.com/prometheus-community/helm-charts/pull/4151 (same for similar changes below)

The result looks like this:
```diff
<         description: The PersistentVolume claimed by {{ $labels.persistentvolumeclaim }} in Namespace {{ $labels.namespace }} on Cluster {{ $labels.cluster }} is only {{ $value | humanizePercentage }} free.
---
>         description: The PersistentVolume claimed by {{ $labels.persistentvolumeclaim }} in Namespace {{ $labels.namespace }} {{ with $labels.cluster -}} on Cluster {{ . }} {{- end }} is only {{ $value | humanizePercentage }} free.
```

Reply:
Thanks @GMartinez-Sisti, this was unintentional. I'll need to have a look as I get those changes also on main, when running sync_prometheus_rules.py.

A further unintended change is in [prometheus.yaml](https://github.com/prometheus-community/helm-charts/pull/4225/files#r1493836396)

Reply:
I've removed those changes. They are from upstream due to [PR 2339](https://github.com/prometheus-operator/kube-prometheus/pull/2339) as the sync script fetches from main

See also upstreamd diff [1](https://github.com/prometheus-operator/kube-prometheus/pull/2339/files#diff-dc8532ace2a75bfcc266c6d1095a60dd1e2ab848af85816057b52d538614bb16R365) [2](https://github.com/prometheus-operator/kube-prometheus/pull/2339/files#diff-dc8532ace2a75bfcc266c6d1095a60dd1e2ab848af85816057b52d538614bb16R385) [3](https://github.com/prometheus-operator/kube-prometheus/pull/2339/files#diff-dc8532ace2a75bfcc266c6d1095a60dd1e2ab848af85816057b52d538614bb16R407) [4](https://github.com/prometheus-operator/kube-prometheus/pull/2339/files#diff-dc8532ace2a75bfcc266c6d1095a60dd1e2ab848af85816057b52d538614bb16R427) [5](https://github.com/prometheus-operator/kube-prometheus/pull/2339/files#diff-dc8532ace2a75bfcc266c6d1095a60dd1e2ab848af85816057b52d538614bb16R449)